README.md

```diff
@@ -26,7 +26,10 @@ git clone https://gitlab.cern.ch/hh/bbtautau/MulticlassInference.git
 git clone git@github.com:hh-italian-group/HHbtag.git HHTools/HHbtag
 
 # KinFit and Combine packages
-git clone git@github.com:LLRCMS/HHKinFit2.git -b bbtautau_LegacyRun2
+mkdir HHKinFit2
+cd HHKinFit2
+git clone git@github.com:jonamotta/HHKinFit2.git -b bbtautau_Run2DNNtraining
+cd ..
```

Comment:
@jonamotta why dont' you push the branch `bbtautau_Run2DNNtraining` directly to `git@github.com:LLRCMS/HHKinFit2.git`? so it stays in a "central repository" and the instructions can simply change to this: 
```suggestion
git clone git@github.com:LLRCMS/HHKinFit2.git -b bbtautau_Run2DNNtraining
```

Reply:
of course if you do this you will have to change all the places below where you have a double `HHKinFit2/HHKinFit2/`

Reply:
I agree on putting it in the central repository not to have personal branches involved (I did not already do it because I do not have the right permissions to push to the HHKinFit repo)
BUT
on the second point (if I understand what you mean) you are wrong because the fact of having `HHKinFit2/HHKinFit2/` does not depend on the fact that I am using a personal branch but on how the package needs to be scrammed and how the HHKinFit code is written

Reply:
to be clearer: this would be the change, not the one you suggested, otherwise the module would not work

```suggestion
mkdir HHKinFit2
cd HHKinFit2
git clone git@github.com:LLRCMS/HHKinFit2.git -b bbtautau_Run2DNNtraining
cd ..
```

Reply:
As agreed offline, I will push the changes of HHKinFit from my personal repo to the central LLRCMS repo and use that one in the instructions.
Still the positioning `HHKinFit2/HHKinFit2/` will remain valid